### ISSUE TITLE
Provide path to .env file at build time

### ISSUE THREAD

**Issue by SharbelOkzan** [ByneappLLC/flutter_config#66]
**Motivation:**
Facilitate working with flavors.

**Goal:**
Have the possibility to pass the path to the .env file with `flutter build` and `flutter run`. It will still default to /.env if not provided.
Users may utilize this feature by having multiple env files and executing:
```
flutter run --flutter-config-path=/.env.stage
# or
flutter run --flutter-config-path=/.env.prod
```

Does that exist? If not, is it theoretically possible?

**Comment by dougalg** [comment]
Yes, this seems like really useful functionality to me. If this is interesting to the maintainers, maybe I could try making a PR
